src/esp/assets/Attributes.h

```diff
@@ -20,34 +20,23 @@ namespace esp {
 namespace assets {
 
 /**
- * @brief Specific Attributes instance which is constructed with a base set of
- * physics object required attributes
+ * @brief base attributes object holding attributes shared by all
+ * PhysicsXXXAttributes objects; Is abstract - should never be instanced
```

Comment:
I had to remove it - i was unable to get pybind to play nicely with the idea of concrete class -> abstract class -> concrete class, and so, while AbstractPhys(ics)Attributes is abstract in essence, i made it concrete so that it could work with pybind.